### PR TITLE
Zos implementation

### PR DESCRIPTION
**Pull request by ewilz**
@mikec did a sizable portion of this work on my computer :D lol

### CODE REVIEW

**Comment by mikec** [comment]
@emaG3m lol you didn't have to credit me but thanks :)

File: ethpm.json [hunk 1]
```diff
@@ -18,10 +18,8 @@
     "voting"
   ],
   "dependencies": {
-    "tokens": "1.0.0",
     "dll": "1.0.4",
-    "attrstore": "1.0.0",
-    "zeppelin": "1.3.0"
+    "attrstore": "1.0.0"
```
Comment (mikec) [inline ethpm.json]: don't think we need `attrstore` or `dll` anymore .. in fact I think we can remove ethpm entirely

**Comment by mikec** [comment]
@emaG3m looks great 👍 ! we can probably remove ethpm, but ok to do that in another PR if you want.

**Comment by ewilz** [comment]
@mikec removed `ethpm.json` just now.

**Comment by ewilz** [comment]
@mikec might wanna take a look at my last few commits so we can merge

**Comment by mikec** [comment]
@emaG3m looks good 👍

**Pull request merged**